SPECS-EXTENDED/nss-mdns/nss-mdns.spec

```diff
@@ -0,0 +1,228 @@
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+
+Name: nss-mdns
+Version: 0.15.1
+Release: 11%{?dist}
+Summary: glibc plugin for .local name resolution
+
+License: LGPLv2+
+URL: https://github.com/lathiat/nss-mdns
+Source0: https://github.com/lathiat/nss-mdns/releases/download/v%{version}/nss-mdns-%{version}.tar.gz
```

Comment:
It seems that we missed to upload the source to the blobstore.
Reference: [link](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=489086&view=logs&j=7909073b-e471-53c1-2c44-627e2efc0564&t=fbeef549-39ab-510c-faa3-78579a516b53)

Reply:
Unfortunately i have no access to the CI build results but i did mention that at the bottom of PR's description e.g. 
> Referring to a docs section i might need some help with that part

So i would like to ask some help with uploading the source

Reply:
Can you please share the sources offline ?